homeassistant/components/zwave_js/sensor.py

```diff
@@ -208,10 +199,15 @@ def async_add_sensor(info: ZwaveDiscoveryInfo) -> None:
         """Add Z-Wave Sensor."""
         entities: list[ZWaveBaseEntity] = []
 
+        if info.platform_data:
+            entity_description_key, unit = info.platform_data
+        else:
+            entity_description_key, unit = None, None
         entity_description = ENTITY_DESCRIPTION_KEY_MAP.get(
-            info.platform_data
+            entity_description_key
         ) or ZwaveSensorEntityDescription("base_sensor")
         entity_description.info = info
```

Comment:
Side note: I think we should remove `info` from the description and pass it separately to the entity. We're modifying the item in the constant container here, which isn't really good, although currently not a bug, since we overwrite the attribute every time.